scripts/update-schema.sh

```diff
@@ -38,7 +39,8 @@ if [ -z $SRCDIR ]; then
   SRC_DIR=`runreadlink -m ${THIS_DIR}/..`
 fi
 
-proto_generator \
+echo  ${SRC_DIR} 
+./proto_generator \
   -path=${SRC_DIR}/v1/yang,${SRC_DIR}/v1/yang/deps \
```

Comment:
Suggest to just make this to be `${GOPATH}/bin/proto_generator` -- or do `export PATH=$GOPATH/bin:$PATH` beforehand so the script is not contextual based on the directory that it is run from.